bin/civi-download-tools

```diff
@@ -164,18 +161,6 @@ function show_command() {
 }
 
 ###############################################################################
-## Debian.org's NodeJS package uses a non-standard name for the node binary.
-## If necessary, setup an alias for the standard name.
-function nodejs_debian_workaround() {
```

Comment:
In the past, we had repeated support issues from Debian/Ubuntu users. It's not a matter of design-aesthetics (by which this function is ugly and properly redflagged as `_workaround`) -- it's a matter of pain-avoidance.

If you have the perspective that it's normal pull in a pre-specified stack (e.g. `--full` or `docker` or `vagrant`), then this wouldn't be necessary. (You could choose full-stacks that provide a normal `node` binary.) But IMHO it's quite useful to be able to run this stuff on different stacks, and sometimes those stacks will be based on Debian/Ubuntu, and often those systems will screw-up the name of the `node` binary.